marble.json

```diff
@@ -1,28 +1,28 @@
 {
-	"response": [
-		{
-			"maintainer": "",
-			"oem": "Poco",
-			"device": "Poco F5",
-			"filename": "crDroidAndroid-14.0-20240602-marble-v10.5.zip",
-			"download": "https://sourceforge.net/projects/crdroid/files/marble/10.x/crDroidAndroid-14.0-20240602-marble-v10.5.zip/download",
-			"timestamp": 1717334132,
-			"md5": "1e9866026c488287aa0686eba7450940",
-			"sha256": "d4afbb21926c5ebf2d5ddfa9efd7d0160173c78a1ace409b2b6d0d93ed56cde0",
-			"size": 1926029208,
-			"version": "10.5",
-			"buildtype": "Monthly",
-			"forum": "https://xdaforums.com/t/rom-14-0-crdroid-10-x-official-pocophone-f5-marble-redmi-note-12-turbo.4606271/",
-			"gapps": "https://sourceforge.net/projects/nikgapps/files/Elite-Releases/crdroid-official/NikGapps-U/20-Apr-2024/NikGapps-crdroid-official-arm64-14-20240420-signed.zip/download",
-			"firmware": "https://xiaomifirmwareupdater.com/firmware/marble/",
-			"modem": "",
-			"bootloader": "",
-			"recovery": "",
-			"paypal": "",
-			"telegram": "https://t.me/Anils_RomHub",
-			"dt": "https://github.com/crdroidandroid/android_device_xiaomi_marble",
-			"common-dt": "",
-			"kernel": "https://github.com/crdroidandroid/android_kernel_xiaomi_marble"
-		}
-	]
-}
+        "response": [
+                {
+                        "maintainer": "Chaitanya",
+                        "oem": "Poco",
+                        "device": "Poco F5",
+                        "filename": "crDroidAndroid-14.0-20240825-marble-v10.7.zip",
+                        "download": "https://sourceforge.net/projects/crdroid/files/marble/10.x/crDroidAndroid-14.0-20240825-marble-v10.7.zip/download",
+                        "timestamp": 1724549911,
+                        "md5": "f61e3a4d836b958695de34185569fb28",
+                        "sha256": "ec6b752379ec88d82e058de4893b9dae964779877fd2aa4815c467bfc4a0f024",
+                        "size": 1894275402,
+                        "version": "10.7",
+                        "buildtype": "Monthly",
+                        "forum": "https://xdaforums.com/t/rom-android-14-official-redmi-note-12t-poco-f5-crdroid-v10-x.4688667/",
+                        "gapps": "https://nikgapps.com/crdroid-official",
+                        "firmware": "https://xiaomifirmwareupdater.com/firmware/marble/",
+                        "modem": "",
+                        "bootloader": "",
+                        "recovery": "https://t.me/chaitanyabuilds/131906",
```

Comment:
don't host on telegram
needs to be able to download even without account

Reply:
oke